modules/streams/src/main/scala/dev/profunktor/redis4cats/streams/streams.scala

```diff
@@ -49,3 +50,6 @@ trait Streaming[F[_], K, V] {
       count: Option[Long] = None
   ): F[XReadMessage[K, V]]
 }
+object Streaming {
+  type Fs2Streaming[K, V] = Streaming[fs2.Stream[IO, *], K, V]
```

Comment:
Since this type alias is only being used in the tests, can you move it to the Suite? Unrelated to your PR, I never liked the naming of this interface anyway 😄 The whole streaming stuff needs some rethinking...

Reply:
great idea 👍